pkg/trace/testutil/span.go

```diff
@@ -393,26 +393,3 @@ func GetTestSpan() *pb.Span {
 	traceutil.ComputeTopLevel(trace)
 	return trace[0]
 }
-
-// TestSpan returns a fix span with hardcoded info, useful for reproducible tests
-func TestSpan() *pb.Span {
```

Comment:
Any reason to remove this? Unused?

Reply:
Ah I'll update the description - yeah in modifying this code I noticed that this function was unused so removed it.